tyk/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "display_name": "Datadog-tyk-integration",
+  "public_title": "Datadog-tyk integration",
+  "author": "Martin Buhr",
+  "maintainer": "yaara@tyk.io",
+  "manifest_version": "1.0.0",
+  "name": "tyk",
+  "metric_prefix": "tyk.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Datadog-tyk integration",
+  "description": "Datadog-tyk integration using StatsdogD",
+  "guid": "7ba25c8a-ca29-4892-bdf6-750e1049d30a",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": true,
+  "integration_id": "tyk",
+  "assets": {
+    "dashboards": {
+      "Tyk Analytics Canvas": "assets/dashboards/tyk_analytics_canvas.json"
+    },
+    "configuration": {
+      "spec": "assets/configuration/spec.yaml"
+    },
```

Comment:
```suggestion
```

Reply:
Sorry - what are you suggesting here @hithwen?